app/src/js/AstroMap.js

```diff
@@ -89,19 +89,29 @@ export default L.Map.AstroMap = L.Map.extend({
    * @param {List} center - Center of map based off of projection.
 ]   */
```

Comment:
I am going to approve but out curiosity, what's this end bracket for?

Reply:
I think it is just a typo